packages/now-build-utils/test/unit.builds-and-routes-detector.test.ts

```diff
@@ -1105,7 +1145,19 @@ it('Test `detectRoutes` with `featHandleMiss=true`', async () => {
       builders!,
       featHandleMiss
     );
-    expect(defaultRoutes).toStrictEqual(expectedRoutes);
+    expect(defaultRoutes).toStrictEqual([
+      { handle: 'miss' },
+      {
+        src: '^/api/([^\\/]+)$',
+        dest: '/api/[endpoint].js?endpoint=$1',
+        check: true,
+      },
```

Comment:
If `cleanUrls` is false, we need to add `{ "src": "/api/(.+)\\.\\w+", "dest": "/api/$1", "check": true }` here

Reply:
At this time, I'm not renaming output files so the routes will need to remain here.

Reply:
Thanks, I added a condition for `cleanUrls`.

- If `true` it will add a redirect.
- If `false, it will add the api route above.

Anything else I'm missing?

Reply:
That's right. We still need to remove the useless public dir rewrite

Reply:
@dav-is Good point, I removed it!